Samples/WindowsML/cs/CSharpConsoleDesktop/CSharpConsoleDesktop/CSharpConsoleDesktop.csproj

```diff
@@ -4,6 +4,8 @@
     <OutputType>Exe</OutputType>
     <WindowsPackageType>None</WindowsPackageType>
     <SelfContained>True</SelfContained>
+    <RuntimeIdentifier Condition="'$(Platform)' == 'x64'">win-x64</RuntimeIdentifier>
```

Comment:
I haven't seen RuntimeIdentifier be conditional before - it's normally specified as a set, like:

```xml
<RuntimeIdentifiers>win-x64;win-arm64</RuntimeIdentifiers>
```

Do you have to constrain it by platform?

Reply:
Yeah, good observation- I'm used to seeing them declared that way as well. And that is actually the approach that I tried originally, but it did not resolve the problem I'm seeing in the build pipeline (I got back essentially the same error where inapplicable RuntimeIdentifier / Platform values are being paired up, e.g. arm64 and win-x64). It seems like the build is trying to produce a cross-product of the combinations when in reality only two pairings are valid. I couldn't figure out how to prevent that from happening- so, definitely open to feedback on how to write this more idiomatically while ensuring the right combinations are used.